vue-frontend/src/components/partials/NewHeader.vue

```diff
@@ -131,36 +149,55 @@
-                currentRoutePath.includes(navbar.url)
-                  ? ''
-                    ? ''
-                    : 'after:tw-w-full after:tw-scale-x-0 after:tw-scale-x-100'
-                  : '',
+                    ' after:tw-absolute after:tw-top-[27px] after:tw-bottom-0 after:tw-left-0 after:tw-w-full after:tw-h-[3px]  after:tw-bg-gradient-[90deg] after:tw-from-[#f2709c] after:tw-to-[#ff9472] after:tw-origin-bottom-left after:tw-duration-300 after:tw-scale-x-0 hover:after:tw-scale-x-100 hover:after:tw-origin-bottom-left hover:tw-bg-clip-text hover:tw-bg-gradient-[270.11deg] hover:tw-from-[#ff9472] hover:tw-to-[#f2709c] hover:tw-text-transparent',
               ]"
             >
               <span
-                :class="[
-                  currentRoutePath == navbar.url
-                    ? 'v2-canopas-gradient-text'
-                    : '',
-                ]"
+                @click="
+                  navbar.name == 'Contribution'
+                    ? (showContributionMenu = !showContributionMenu)
+                    : ''
+                "
                 class="tw-flex tw-mt-[0.1rem]"
                 >{{ navbar.name }}</span
               ></router-link
             >
+
             <a
               v-else
-              class="tw-inline-block tw-relative tw-mr-[20px] sm:tw-mr-[30px] after:tw-absolute after:tw-w-full after:tw-h-[3px] after:tw-top-[27px] after:tw-bottom-0 after:tw-left-0 after:tw-bg-gradient-[90deg] after:tw-from-[#f2709c] after:tw-to-[#ff9472] after:tw-origin-bottom-left after:tw-duration-300 after:tw-scale-x-0 hover:after:tw-scale-x-100 hover:after:tw-origin-bottom-left hover:tw-bg-clip-text hover:tw-bg-gradient-[270.11deg] hover:tw-from-[#ff9472] hover:tw-to-[#f2709c] hover:tw-text-transparent"
+              class="tw-inline-block tw-relative tw-mr-[20px] sm:tw-mr-[30px] after:tw-absolute after:tw-top-[27px] after:tw-bottom-0 after:tw-left-0 after:tw-w-full after:tw-h-[3px] after:tw-bg-gradient-[90deg] after:tw-from-[#f2709c] after:tw-to-[#ff9472] after:tw-origin-bottom-left after:tw-duration-300 after:tw-scale-x-0 hover:after:tw-scale-x-100 hover:after:tw-origin-bottom-left hover:tw-bg-clip-text hover:tw-bg-gradient-[270.11deg] hover:tw-from-[#ff9472] hover:tw-to-[#f2709c] hover:tw-text-transparent"
               :href="navbar.url"
               :target="navbar.target"
               >{{ navbar.name }}</a
             >
+            <ul
+              :class="
+                showContributionMenu && navbar.name == 'Contribution'
+                  ? 'tw-mt-8 tw-w-full tw-px-6 tw-bg-white'
```

Comment:
![image](https://github.com/canopas/canopas-website/assets/98794636/d340d7b3-dc00-4c26-8627-502eeeb95192)

Reply:
text is not changing on hover.